.storybook/main.ts

```diff
@@ -59,9 +59,24 @@ const config: StorybookConfig = {
       // accept Chakra props but this makes it so that the Chakra props
       // are just hidden in Storybook.
       propFilter: (prop) => {
-        // This is a bit of a "hack" but it tells Storybook that the prop
-        // is defined in a DS component. If we define it, then display it
-        // in the `ArgTypes` table in a component Storybook document page.
+        /** @Info: The following code would allow for typescript `type`
```

Comment:
This is out of scope for this PR but I want you to keep it here commented out. We can make a new ticket to uncomment it and test it on all components so it can be tested separately from the `SocialMediaLinks` component. I would prefer that since this configuration affects all components. Again, leave it here to address later.

Reply:
Yes that's what I thought too. Would be cool if it worked but needs definitely more testing.